CIP-0001/README.md

```diff
@@ -242,40 +250,48 @@ This must be subdivided into two sub-sections:
-- c) defining boundaries within their project for which the CIP process does apply;
-- d) writing CIPs for significant changes introduced in their projects when it applies.
+- allocating time to **review** proposals from actors of the community when solicited by editors (i.e. after one first round of reviews);
+- defining additional rules and processes whereby external actors can engage with their project as part of the CIP process;
+- defining boundaries within their project for which the CIP process does apply;
+- establishing points of contact and any designated reviews for a category;
+- agreeing upon how proposals move from the state of idea (i.e. CIP) to actual implementation work;
+- writing CIPs for significant changes introduced in their projects when it applies.
 
-Enlisting for the CIP process happens by creating a CIP. That CIP must encapsulate the information listed above, as well as any other pieces of information deemed helpful to future authors. Of course, only team members of a target project can author such a proposal.
+Any guidelines for this cooperation should be described by a dedicated CIP whenever possible.  When such a CIP is posted or supersedes another one, it will be entered into the above table in the Categories section.  Participants of enlisted categories should follow the requirements outlined in that CIP and should update such proposals whenever these requirements or relationships change.
 
-> **Warning** A positive review by the maintainers of a project does not constitute a commitment to implement the CIP. It is still the CIP author's responsibility to create an implementation plan and identify implementors. The maintainers of the project may volunteer to participate in implementation, but also may not. Projects' maintainers ultimately define how a proposal can move from the state of idea (i.e. CIP) to actual implementation work. We, however, expect each team that enlists in the CIP process to provide clarity on these elements as they enlist.
+> **Warning** A positive review by any enlisted project representative does not constitute a commitment to implement the CIP. It is still the CIP author's responsibility to create an implementation plan and identify implementors.
 
-Editors occasionally invite project maintainers to speak during review meetings and solicit them for ultimate approvals of proposals affecting a project under their authority. Said differently, CIPs that concern (part of) an enlisted project will only be merged after explicit acceptance of the enlisted reviewers.
+Editors occasionally invite representatives from enlisted categories to speak during review meetings and solicit them for ultimate approvals of proposals in their area of expertise.
 
 > **Note** Optionally, projects may show their enlisting using the following badge on their introductory README: ![](https://github.com/cardano-foundation/CIPs](https://raw.githubusercontent.com/cardano-foundation/CIPs/master/.github/badge.svg)
```

Comment:
```suggestion
**Note** Optionally, projects may show their enlisting using the following badge on their introductory README: ![](https://github.com/cardano-foundation/CIPs](https://raw.githubusercontent.com/cardano-foundation/CIPs/master/.github/badge.svg)
```

This part doesnt render properly, maybe we should take out the `>`

Reply:
sure; let's try it.  That code is an eyesore everywhere and leaves us looking over our shoulder on derived sites like `cips.cardano.org` and the Dev Portal.  It does look dubious as written but I'd never suggested a correction since I didn't really know how the embed was supposed to look.  If still broken anywhere after the merge, we can move it to a Gist for further testing, with any project that might be using this badge (?), and then fix it in another PR.